subprojects/docs/src/samples/testKit/gradleRunner/manualClasspathInjection/groovy/build.gradle

```diff
@@ -20,8 +20,12 @@ repositories {
 task createClasspathManifest {
     def outputDir = file("$buildDir/$name")
 
-    inputs.files sourceSets.main.runtimeClasspath
-    outputs.dir outputDir
+    inputs.files(sourceSets.main.runtimeClasspath)
+        .withPropertyName("runtimeClasspath")
+        .withPathSensitivity(PathSensitivity.RELATIVE)
+        .withNormalizer(ClasspathNormalizer)
```

Comment:
@lptr, does it make sense to add a classpath normalizer here?
I didn't see any such example in our codebase

Reply:
It does, and it makes the `.withPathSensitivity(PathSensitivity.RELATIVE)` redundant.

Reply:
Did you check the documentation text from where we link to this?

Reply:
Checked the javadocs that didn't answer the above question.
Do you mean any specific doc I could check nex time?
Removed `withPathSensitivity` from here

Reply:
Sorry, what I meant was: could you check the doc that links to this sample, and see if it needs adjusting?

To answer your question all should be documented (albeit badly) here: https://docs.gradle.org/current/userguide/more_about_tasks.html#sec:task_input_output_annotations

Reply:
Did you get to look at `test_kit.adoc`?

Reply:
We talked about this and the doc has been checked.